docs/app/Examples/collections/Form/Validation/FormValidationExamples.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/jsx-curly-spacing */
```

Comment:
This rule has been turned off entirely for the project.  I do not think it works correctly with the spread operators.  You should be able to remove this after rebasing to master.